### ISSUE TITLE
Add PNG dWLm (Diffuse white luminance metadata ) chunk to conform to ISO 22028-5

### ISSUE THREAD

**Issue by digitaltvguy** [w3c/PNG-spec#390]
ISO 22028-5 High dynamic range and wide colour gamut encoding for still images (HDR/WCG) identifies a very important parameter (Diffuse white luminance metadata ).

This metadata allows us to determine at which point highlight compression might occur during HDR to SDR conversion.

**Comment by palemieux** [comment]
What is the relationship between _diffuse white luminance metadata_ and _reference white_ specified in [Report ITU-R BT.2408](https://www.itu.int/pub/R-REP-BT.2408)?

**Comment by ProgramMax** [comment]
It sounds like ISO 22028-5 will be paywalled when published.
Is ITU-R BT.2408 effectively the same (and not paywalled)?

**Comment by svgeesus** [comment]
![Image](https://github.com/user-attachments/assets/5544c622-2058-4051-be74-d1615e763a9b)

**Comment by svgeesus** [comment]
> What is the relationship between _diffuse white luminance metadata_ and _reference white_

What was previously called _Reference White_ is now renamed to _HDR Reference White_ for clarity.

**Comment by svgeesus** [comment]
So in practice, the metadata is a way to override the definition in BT.2048, ie when the luminance is **not** 203

**Comment by leo-barnes** [comment]
> So in practice, the metadata is a way to override the definition in BT.2048, ie when the luminance is **not** 203

Yep, pretty much. IIRC 203 nits is just a default, but before there was not really any way to signal anything other than the default.

**Comment by ProgramMax** [comment]
I believe we discussed long ago renaming this chunk to rWTm for "Reference WhiTe ....Metadata?".
I forget what the M stood for. Does anyone remember?

**Comment by leo-barnes** [comment]
In HEIF, the box is:
```
class NominalDiffuseWhiteBox extends ItemFullProperty('ndwt', 0, 0)
```

**Comment by Filoppi** [comment]
Thanks. Adding some kind of "Paper White" or "Diffuse White" value for HDR content is extremely important so that viewers could re-map the source white to the local display white, instead of viewing content that might be too dark or bright for you. This is basically biggest problem with HDR so far.

**Comment by svgeesus** [comment]
> I believe we discussed long ago renaming this chunk to rWTm for "Reference WhiTe ....Metadata?".

Since ISO 22028-5 renamed what was previously "Reference White" to "HDR Reference White" I feel we should follow that terminology and this suggest `hRWl` for **H**DR **R**eference **W**hite **L**evel. Because the value is a luminance level.

**Comment by svgeesus** [comment]
> What is the relationship between _diffuse white luminance metadata_ and _reference white_ specified in [Report ITU-R BT.2408](https://www.itu.int/pub/R-REP-BT.2408)?

In the latest (March 2026) BT.2408 this has been renamed to **HDR Reference White**, same as ISO 22028-5.

Given that BT.2408 is _freely available_ and the definition is identical to ISO 22028-5 then I believe from a W3C references perspective we can add both as normative references provided we note that the definitions are identical.
